src/less/filter-menu-button/base/filter-menu-button.less

```diff
@@ -0,0 +1,247 @@
+div.filter-group {
+    .filter-group();
+}
+
+span.filter-menu-button {
+    .filter-button-sibling-spacing();
+
+    display: inline-block;
+    position: relative;
+}
+
+button.filter-menu-button__button {
+    .filter-button-base();
+
+    background-color: @filter-button-background-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-hover-background-color;
+    }
+}
+
+button.filter-menu-button__button[aria-expanded="true"] {
+    background-color: @filter-button-hover-background-color;
+}
+
+.filter-menu-button__button-cell {
+    display: flex;
+
+    & > svg.icon {
+        color: @filter-menu-button-chevron-color;
+        height: 7px;
+        width: 12px;
+    }
+
+    & > span.icon {
+        .background-icon-base-mixin();
+        .background-icon-image-mixin('chevron-down', 12px, 7px);
+    }
+
+    svg.icon,
+    span.icon {
+        align-self: center;
+        margin-left: 8px;
+    }
+}
+
+.filter-menu-button__button-text {
+    display: inline-block;
+    max-width: 238px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.filter-menu-button__button[aria-expanded="true"] .filter-menu-button__button-cell > .icon {
+    transform: rotate(180deg);
+}
+
+button.filter-menu-button__button[aria-pressed="true"] {
+    .filter-button-selected();
+
+    background-color: @filter-button-selected-background-color; // should we just lighten @color-b4;
+    color: @filter-button-selected-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-selected-hover-background-color; // should we just lighten @color-b4;
+    }
+}
+
+button.filter-menu-button__button[aria-expanded="true"][aria-pressed="true"] {
+    background-color: @filter-button-selected-hover-background-color; // should we just lighten @color-b4;
+}
+
+button.filter-menu-button__button[disabled],
+button.filter-menu-button__button[aria-disabled="true"],
+button.filter-menu-button__button[aria-pressed="true"][disabled],
```

Comment:
Do we need all the extra selectors? I would think [disabled] and [aria-disabled=true] are good enough. Why need the aria-pressed too?

Reply:
Those are the different variations of states of the button. If I leave one out then we have a situation where it doesn't properly style for that combination of properties.

Reply:
Won't button.filter-menu-button__button[disabled] selector be applied when aria-pressed=true? Or am I missing something. If so, then we don't need [aria-pressed=true][disabled].

Reply:
There are conflicting styles with when `[aria-pressed="true"]` and just `[disabled]` given the order of the CSS in the file (avoiding the `no-descending-specificy` rule we have set up).

Reply:
But ... I will look into simplifying this.

Reply:
Got it. Just use `!important`!....

Just kidding, that's what I assumed but just wanted to double check!